Makefile

```diff
@@ -73,7 +73,7 @@ rpm-release: srpm-release
 	mock -r $(MOCK_CONFIG) --resultdir BUILD/RPM -D "rel_build 1" --rebuild BUILD/SRPM/python-aexpect-$(VERSION)-*.src.rpm
 
 check: clean
-	inspekt checkall --disable-lint W1618,R0205
+	inspekt checkall --disable-lint W1618,R0205,W0707,R1725
```

Comment:
@clebergnu Will write this down in the issue you created but we can always prioritize fixing more than the pylint disable cases mentioned there.